app/components/avatar-list.tsx

```diff
@@ -1,10 +1,23 @@
 import { take, slice } from "lodash"
 
 export type AvatarListProps = {
-  list?: { name: string; initials: string; pictureUrl?: string }[]
+  list?: AvatarType[]
+  handleOnClick: () => void
 } & JSX.IntrinsicElements["div"]
 
-function getColor(string: string) {
+export type AvatarProps = {
+  avatar: AvatarType
+  displayName?: Boolean
+  big?: Boolean
```

Comment:
Please make this an enum, like `size: "big", "regular"`. It helps reading the component and also helps adding future sizes.